src/gwc/pom.xml

```diff
@@ -85,6 +85,10 @@
       <groupId>org.geowebcache</groupId>
       <artifactId>gwc-diskquota-jdbc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.geowebcache</groupId>
+      <artifactId>gwc-aws-s3</artifactId>
```

Comment:
Just realised... if S3 becomes an extension we would need to change the singleton design anyway... the S3 gui would also need to be separated and the blobstore types (with their own gui pages) would need to be extendible, with indeed something like spring, and we couldn't use the read-only singleton obviously.